packages/local_auth/lib/auth_strings.dart

```diff
@@ -15,35 +15,35 @@ import 'package:intl/intl.dart';
 /// Provides default values for all messages.
 class AndroidAuthMessages {
   const AndroidAuthMessages({
-    this.fingerprintHint,
-    this.fingerprintNotRecognized,
-    this.fingerprintSuccess,
+    this.biometricHint,
```

Comment:
This is a breaking change. Could you please add it to the CHANGELOG?

Reply:
Noted to changelog.